qemu/tests/qemu_img_extent_size_hint.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+from virttest import data_dir, utils_misc
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    qemu-img measure a new image.
+
+    1. create an image with default extent_size_hit
+    3. check it in qemu-img info
+    4. create an image with extent_size_hit in range [1M 3.5G]
+    5. check it in qemu-img info
+    6. create an image with extent_size_hit=0
+    7. check it in qemu-img info
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _image_create():
+        """Create an image."""
+        image_name = params.get("images")
+        img_param = params.object_params(image_name)
+        img = QemuImg(img_param, data_dir.get_data_dir(), image_name)
+        img.create(img_param)
+        return img
+
+    def ck_extent_size_hint(img, expect, v_set):
+        """Check extent_size_hint in qemu-img info"""
+        parttern = params.get("esh_pattern")
+        output = img.info()
+        test.log.info("Check the extent size hint from output")
+        es_hint = re.findall(parttern, output)
+        if es_hint:
+            if es_hint[0] != expect:
+                test.fail("Extent_size_hint %s is not expected value %s",
```

Comment:
test.fail("Extent_size_hint %s is not expected value %s" % (es_hint, expect)), we use test.log.xxx this way